lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/IntegrationStrategy.cxx

```diff
@@ -202,6 +202,12 @@ void IntegrationStrategy::load(Advocate & adv)
   ProjectionStrategyImplementation::load(adv);
 }
 
+/* Selection history accessor */
+Collection<Indices> IntegrationStrategy::getSelectionHistory(Collection<Point> & coefficientsHistory) const
+{
+  coefficientsHistory = Collection<Point>();
```

Comment:
Coupled with CleaningStrategy, an IntegrationStrategy may produce a selection history. It can be postpone to another PR of course.

Reply:
yes, also we can do cleaning+lars ! :)

Reply:
So is for the `LinearModelStepwiseAlgorithm`, which can be used for a PCE too.